test/plugins/pylint_test.py

```diff
@@ -122,7 +122,7 @@ def test_invalid_python():
     message = messages.pop()
     # Pylint 1.x says the source is <string>, Pylint 2.x says <unknown>
     message = message.replace("<string>", "XXX").replace("<unknown>", "XXX")
-    assert message == "1:syntax-error:invalid syntax (XXX, line 1)"
+    assert message == "1:syntax-error:Parsing failed: 'invalid syntax (XXX, line 1)'"
```

Comment:
Error message format updated in https://github.com/PyCQA/pylint/commit/36a6723ce6b15e8a466a300b6594d6828305fd67#diff-e7570dac4f212ecd0ade4b2d5f48c31272f5650f189c2b93349e9b5456ff8b3cL1